Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Tools/DCC/Blender/AddOns/SceneExporter/utils.py

```diff
@@ -127,23 +157,40 @@ def loop_through_selected_materials(texture_file_path):
                         bpy.data.images[img.image.name].filepath = str(Path(texture_file_path).joinpath(build_string))
                     
                     bpy.data.images[img.image.name].save()
-                full_path = bpy.path.abspath(img.image.filepath, library=img.image.library)
+                full_path = Path(bpy.path.abspath(img.image.filepath, library=img.image.library))
                 base_name = Path(full_path).name
+                if not full_path.exists(): # if embedded path doesnt exist, check current folder
+                    #full_path = str(Path(bpy.data.filepath).parent.joinpath(base_name))

```

Comment:
Is there a reason for leaving this in when it is inactive?

Reply:
Opps, I will take this out. Thanks.